projects/js-packages/components/components/donut-meter/index.tsx

```diff
@@ -71,14 +99,17 @@ const DonutMeter: React.FC< DonutMeterProps > = ( {
 			? 'true'
 			: 'false';
 
+	const finalClassName = `donut-meter ${ className ? className + ' ' : '' }${
+		type ? 'is-' + type + ' ' : ''
+	} ${ ! type && useAdaptiveColors ? 'is-' + getAdaptiveType( percentage ) + ' ' : '' }`.trim();
```

Comment:
I love this combination concept, which occurred to me when I thought of this donut component. 👍🏼

Reply:
🙇